### ISSUE TITLE
Collaboration with Rivets useful?

### ISSUE THREAD

**Issue by dev-rke** [lega911/angular-light#199]
Hi @lega911,

i just want to let you note, that there is a similar project called [rivetsjs](https://github.com/mikeric/rivets/), which is also written in coffeescript, even they work in an ES6 branch.

It has less functionality and flexibility than alight, but it seems better structured and documented and therefore easier to learn. But regarding performance alight is far better.
 
Maybe it might be useful to enrich rivets with some features from alight or to simplify/improve some things in alight by analyzing implementations in rivets.
I think a collaboration between both projects will let them benefit both.

Its just an idea, have a look at it and if it does not fit to your ideas, just ignore me. ;-)

**Comment by lega911** [comment]
thank you for link, I will read it,
aslo I have plans to simplify alight

**Comment by lega911** [comment]
@dev-rke
yes, rivets is not rich in features, even { a + a } or { foo() } don't work.
it has another type of change detector - observers with properties, so it has own collection of disadvantages (problems with observing, it replaces data to properties - spoils source data, works slower, ...). it's closer to vue.js

besides of features, every framework has own problems (because of js), so a developer should choose what problems to have

**Comment by lega911** [comment]
addition:
`rv-if="auction.timeLeft | lt 120"`
vs
`al-if="auction.timeLeft < 120"`

`rv-text="event.duration | call event.start event.end"`  <- it seems you need write all dependencies manually :(
vs
`{{ event.duration() }}`

give me an example where rivets looks better

**Comment by dev-rke** [comment]
Hi,

i did not say it is better.
It has just less features, but these are a bit better documented in the Docs.
The most important part for such a Framework is Template binding and UserControl binding. Both provide them.
